web/src/generatedTypes/AtlascopeTypes.ts

```diff
@@ -452,6 +452,15 @@ export interface Pin {
   child?: number;
 }
 
+export interface VisOption {
+  value: string,
+  text: string,
+  data: Dataset,
+  visFunc: Function,
+  options: string[],
+  availableOptions: object[],
+}
```

Comment:
Was this type automatically generated? I don't remember where it would come from on the back end. If not, re-running the `generateTypes` script will delete this, so I think this type should be defined elsewhere.